convert2rhel/systeminfo.py

```diff
@@ -188,6 +189,16 @@ def modified_rpm_files_diff(self):
             self.logger.info(
                 "Comparison of modified rpm files from before and after the conversion:\n%s" % modified_rpm_files_diff)
 
+    @staticmethod
+    def has_rpm(name):
```

Comment:
This feels more like it belongs in `utils` since it's more a utility than systeminfo function

Reply:
my idea was to write something like 
```python
if system.has_rmp:
    blabla
```
and I found system_info, which is close. But I don't any strong arguments for both solutions.
WDYT @oamg/convert2rhel

Reply:
Having it here seems fine. I'd just rename it to `is_rpm_installed`. Because `if system_info.has_rpm("bogus")` does not convey the same as `system.has_rpm`.

Reply:
@SpyTec I would go with @bocekm solution.

Reply:
SGTM